app/animal-management-naive-dont-copy/layout.tsx

```diff
@@ -22,5 +16,5 @@ export default async function AnimalManagementNaiveDontCopyLayout(
     redirect(`/login?returnTo=${headersList.get('x-pathname')}`);
   }
 
-  return props.children;
+  return;
```

Comment:
<img width="901" alt="image" src="https://github.com/upleveled/next-js-example-winter-2023-vienna-austria/assets/61600906/a6a60ac6-1d5b-4dc9-b86a-9bbc00b07c42">

This is the error i am receiving but since is not my code and also not part of this PR ill revert

Reply:
fd51ada